casper/src/main/scala/coop/rchain/casper/genesis/contracts/Rev.scala

```diff
@@ -18,6 +18,8 @@ class Rev[A](
   private val minimumBond = posParams.minimumBond
   private val maximumBond = posParams.maximumBond
 
+  final val path = "<synthetic in Rev.scala>"
```

Comment:
Why isn't this optional?

Reply:
Because you want to know even if it's synthetic (generated) source

Reply:
I do, anyway

Reply:
Know what exactly? 
The characters currently residing under the `path` val express the fact that for this particular class a `path` doesn't exist. This can be expressed isomorphically with a `None`

Reply:
the path, what else? this field is purely for debuggability purposes. It's not iso with None, as once s/b generates more .rho in scala, they need to provide the path.